app/__init__.py

```diff
@@ -108,10 +110,23 @@ def download_csv():
                     keys=keys
                 )
 
+            columns = list(keys)
+            columns[-1] = "lnglat"
+            column_names = list(keys)
+            column_names[-1] = "longitude"
+            column_names.append("latitude")
```

Comment:
@christian-rli - Do you know which geographical unit/system of reference I can add after longitude/latitude ? (ie 4326, or something else than just longitude, latitude?)

Reply:
I am not sure if I understand the question correctly. What do you need to specify and what for? 4326 is the EPSG code for WGS 84, which is what leaflet / the webmap uses. 3857 is what's on the database and which we translate to WGS 84 before displaying it.

Reply:
My question is: does writing "latitude WGS 84" or "latitude 3857" make sense ? And is is the conventional way to do it?. I want to be as precise as possible with the geographical unit.

Reply:
To be honest I don't know what the best way to put it grammatically is, but specifying "latitude on WGS 84" makes total sense. WGS 84 is often assumed when no specification is given, but that should be considered bad practice. So specifying the coordinate reference system gets my thumbs up :)

Reply:
(not sure about conventional ways. in files with geo-data it usually comes with the metadata)

Reply:
```suggestion
            column_names[-1] = "longitude on WGS 84"
            column_names.append("latitude on WGS 84")
```